docker/ubuntu-12.04/Dockerfile

```diff
@@ -1,13 +1,15 @@
 # FpDebug with Valgrind 3.7 on Ubuntu 12.04
 FROM ubuntu:12.04
 
+RUN sed -i -re 's/([a-z]{2}\.)?archive.ubuntu.com|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list
+
 RUN apt-get update && apt-get install -y \
   git \
   bzip2 \
   build-essential \
   m4
 
-RUN git clone https://github.com/fbenz/FpDebug.git
```

Comment:
Please revert this change. It makes sense on your fork but not when merging back.

Reply:
Done. I should have used a separate branch.